localization/english/ancient_magic_education_l_english.yml

```diff
@@ -1,17 +1,17 @@
 ﻿l_english:
-#traits
- trait_education_arcane_1:0 "Apprentice wizard"
- trait_education_arcane_1_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is an apprentice practitioner of the arcane arts. Though a wizard by training, due to some defect, whether that defect be esoteric or mundane, [ROOT.Char.GetHerHis] attempts at spell casting are childish, often surpassed by even the most inexperienced of apprentices."
- trait_education_arcane_2:0 "Mediocre wizard"
- trait_education_arcane_2_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is a mediocre practitioner of the arcane arts. Though a wizard by training, due to some defect, whether that defect be esoteric or mundane, [ROOT.Char.GetSheHe] is a less capable spell caster than others."
- trait_education_arcane_3:0 "Wizard"
- trait_education_arcane_3_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is a well respected practitioner of the arcane arts, capable of creating spells with finesse and ease."
- trait_education_arcane_4:0 "Elite wizard"
- trait_education_arcane_4_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is one of the foremost experts on magic in the world. [ROOT.Char.GetSheHe|U] is a prodigy in both the theory and practical execution of spell casting."
- #events
+ #traits
+ trait_education_arcane_1: "Magic Student"
+ trait_education_arcane_1_desc: "[ROOT.Char.GetFirstNameNoTooltip] attended [ROOT.Char.GetHerHis] lessons, though that's about all that can be said. [ROOT.Char.GetHerHis|U] attempts to recall their studies are laughable, often surpassed by even the most inexperienced of apprentices."
+ trait_education_arcane_2: "Magic Journeyman"
+ trait_education_arcane_2_desc: "[ROOT.Char.GetFirstNameNoTooltip] is a mediocre arcane historian. Though [ROOT.Char.GetSheHe] attended lessons on magical history and theory, [ROOT.Char.GetSheHe] may have gotten less out of them than other students."
+ trait_education_arcane_3: "Magic Expert"
+ trait_education_arcane_3_desc: "[ROOT.Char.GetFirstNameNoTooltip] is a well respected expert on magical theory, capable of holding their own against most magical historians... if society permits."
+ trait_education_arcane_4: "Magical Theorist"
+ trait_education_arcane_4_desc: "[ROOT.Char.GetFirstNameNoTooltip] is one of the foremost experts on magic in the world. [ROOT.Char.GetSheHe|U] is a prodigy in the theory of spell casting, and could become one of the greatest practitioners."
+
  #education
- education_arcane:0 "Arcane Education" 
```

Comment:
why were the 0 deleted ?

Reply:
again i really dont know, im just asking

Reply:
They're only used by Paradox dev tools for debugging purposes. For us, they do nothing but mess with the coloring in our IDEs. I removed them to make it color the keys differently so it would be easy to tell what text is keys and what text is values.

Reply:
ok